internal/client.go

```diff
@@ -139,7 +141,7 @@ type (
 		//	- BadRequestError
 		//	- InternalServiceError
 		//	- WorkflowExecutionAlreadyCompletedError
-		CancelWorkflow(ctx context.Context, workflowID string, runID string) error
+		CancelWorkflow(ctx context.Context, workflowID string, runID string, opts ...Option) error
```

Comment:
unfortunately changes to interfaces are always breaking :\  it's one of the things we need to address in the next api version, since it's so crippling.